ProcessLib/GroundwaterFlowFEM.h

```diff
@@ -111,7 +116,7 @@ class LocalAssemblerData : public LocalAssemblerDataInterface<GlobalMatrix, Glob
 
 private:
     std::vector<ShapeMatrices> _shape_matrices;
-    double _hydraulic_conductivity;
+    std::function<double(void)> _hydraulic_conductivity;
```

Comment:
Definetly. There are already some functions managing the tuple size information; See the [[PL] Remove unused parameter tuple functions.](https://github.com/endJunction/ogs/commit/b313ed5c923465311b5d8f3054674560d009032b).